SPECS/openjdk8/openjdk8.spec

```diff
@@ -113,6 +113,7 @@ unset JAVA_HOME &&
 	--with-freetype-lib=%{_libdir} \
 	--with-stdc++lib=dynamic \
 	--with-native-debug-symbols=none \
+  --with-zlib=system \
```

Comment:
NIT: inconsistent spacing

Reply:
Yuck, this whole file is a mix of both tab and space indenting... fixed this one to match the rest of the command at least.